src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -53,6 +54,7 @@ class BucketAggregationValues extends Component {
           aggName={this.childAgg.aggName}
           childAgg={this.childAgg.childAgg}
           onFilterClicked={onFilterClicked}
+          parentValue={bucket.key}
```

Comment:
Very minor: I think that we tend to keep the `key` naming around.
Shall we call this `parentKey` or `parentBucketKey` instead?